### PR TITLE
New Seeker, HPR, Shotgun. Gun changes.

### PR DESCRIPTION
**Pull request by Viz-Titan**
## About The Pull Request
- Added in the Seeker Rifle, Heavy Pulse Rifle, and the SCL Shotgun(Pulse Shotgun). Tweaked Divet Pistol and Pulse Rifle.
- Added in some diverse gun and ammo variants as well for most of them. 
- Added in some blank files and sounds, along with revising sprite files. May help slightly with future endeavors with guns. 
- Scavbot and Tcomms dish machinery sprites put in. 
- Got rid of the config/admins.txt ported from Daed

## Why It's Good For The Game

More guns, revised guns. 

## Changelog

:cl:
add: Added in guns: Seeker Rifle, Heavy Pulse Rifle, and SCL/Pulse Shotgun
add: Added in various ammo and gun variants for some guns
balance: Tweaked and added to Divet Pistol and Pulse Rifle
imageadd: added scavbot and a tcomms dish machine sprites. Fixed Pulse Rifle sprite.
admin: Removed old Daed Hosts from admins.txt
/:cl:

### CODE REVIEW

**Comment by Viz-Titan** [comment]
Finally got this in

File: necromorphs/code/modules/weapons/guns/divet.dm [hunk 1]
```diff
@@ -1,50 +1,235 @@
+/**
+Divet pistols
+*/
+
 /obj/item/gun/ballistic/automatic/pistol/divet
 	name = "divet pistol"
 	desc = "A Winchester Arms NK-series pistol capable of fully automatic fire."
-	icon = 'necromorphs/icons/obj/weapons/gun.dmi'
+	icon = 'necromorphs/icons/obj/weapons/ds13guns.dmi'
```
Comment (DTraitor) [inline necromorphs/code/modules/weapons/guns/divet.dm:8]: What is the point of renaming the dmi? It is in the dead space folder already

File: necromorphs/code/modules/weapons/guns/divet.dm [hunk 2]
```diff
@@ -1,50 +1,235 @@
...
+	fire_sound= 'sound/ds13/divet_fire.ogg'
+	load_sound = 'necromorphs/sound/weapons/guns/divet_magin.ogg'  //Old noises from 1.0, but weren't actually working and used before.
+	eject_sound = 'necromorphs/sound/weapons/guns/divet_magout.ogg' //They just sound like clockwork cult sounds.
+	suppressed_sound = 'sound/weapons/gun/general/heavy_shot_suppressed.ogg'
+
+/obj/item/gun/ballistic/automatic/pistol/divet/extended
+	mag_type = /obj/item/ammo_box/magazine/divet/extended //Figure out how to keep compatability with normal mags, but use special mag. Spcial amg types may work if this doesn't
+
+/obj/item/gun/ballistic/automatic/pistol/divet/extended
+	mag_type = /obj/item/ammo_box/magazine/divet/extended/expanded
```
Comment (DTraitor) [inline necromorphs/code/modules/weapons/guns/divet.dm:31]: Duplicate code

File: necromorphs/code/modules/weapons/guns/pulse_rifle.dm [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+	caliber = CALIBER_PULSE
+	ammo_type = /obj/item/ammo_casing/caseless/pulse
+	max_ammo = 50 //To low? 65 normal
+
+// /obj/item/ammo_box/magazine/pulse/update_icon_state()
+// 	..()
+// 	if (length(stored_ammo))
+// 		icon_state = "pulse_rounds"
+// 	else
+// 		icon_state = "pulse_rounds_empty"
```
Comment (DTraitor) [inline necromorphs/code/modules/weapons/guns/pulse_rifle.dm:66]: Why did you leave these comments? If the code is not used and you aren't going to implement it in the future - remove it

**Review by DTraitor — CHANGES_REQUESTED** [review]
Requires some changes, also remove empty files, we don't need them

File: necromorphs/code/modules/weapons/guns/divet.dm (hunk 1 above)
Comment (Viz-Titan) [reply necromorphs/code/modules/weapons/guns/divet.dm:8]: image

Comment (Viz-Titan) [reply necromorphs/code/modules/weapons/guns/divet.dm:8]: Was just named "guns" before hand. Made it "ds13guns" to work with the other file names

File: necromorphs/code/modules/weapons/guns/divet.dm (hunk 2 above)
Comment (Viz-Titan) [reply necromorphs/code/modules/weapons/guns/divet.dm:31]: Whoops. Meant for each magtype. 
I'll take a moment to get the mags figured out

**Comment by Viz-Titan** [comment]
Also merge Comet Blaze's PR before mine, and I will tweak mine

**Comment by Viz-Titan** [comment]
Finally figured out how to do it

**Pull request merged**